Vagrantfile

```diff
@@ -25,6 +25,7 @@ Vagrant.configure("2") do |config|
       vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
     end
 
+    config.vagrant.plugins = {"vagrant-env" => {"version" => "0.0.3"}}
```

Comment:
@jerome-caucat please move that line at the top of the `Vagrant.configure(` block.

Reply:
@JMLX42 Since it's my first MR here I have two questions:
1. What is the preferred method to update my branch with master: merge or rebase?
2. Should I squash the commits?

Reply:
> What is the preferred method to update my branch with master: merge or rebase?

IDK. Rebase if possible I guess. But I'm not even sure you actually need to do that.

> Should I squash the commits?

Not necessarily.

Reply:
Ok, I guess it depends on the situation, for example in this case it is cleaner to rebase and squash but in some cases we will want to keep multiple commits so the history is explicit.

Anyway, I've now moved the line at the top of the `Vagrant.configure` block.